tests/unit/test_message_queue.cpp

```diff
@@ -86,10 +86,10 @@ void TestMessageQueue(void)
 
     sMessagePool = &sInstance->Get<ot::MessagePool>();
 
-    for (int i = 0; i < kNumTestMessages; i++)
+    for (auto &m : msg)
```

Comment:
```suggestion
    for (auto &msg : messages)
```

Reply:
`msg` is already used, so I used `m`. 
Changing `msg` to `messages` would bring too much code change. Since it's unittest code (which are not expected to scale), would `m` and `msg` be just fine?

Reply:
`m` is usually a number (just like `n`). Generally, I don't encourage single-char variable names except for `i`, `j`, `m`, `n` (`m`, `n` are numbers).

Reply:
How about renaming `msg` array to `messages` and then use `message` or `msg` here in the iteratiorn loop?